app/controllers/collection_objects_controller.rb

```diff
@@ -98,7 +98,9 @@ def destroy
 
   # GET /collection_objects/list
   def list
-    @collection_objects = CollectionObject.with_project_id(sessions_current_project_id).order(:id).page(params[:page]) #.per(10) #.per(3)
+    @collection_objects = CollectionObject.with_project_id(sessions_current_project_id)
+                            .order(:id)
+                            .page(params[:page]) #.per(10) #.per(3)
```

Comment:
Remind me, is params[:page] "safe"?

Reply:
As far as I know, yes. That is how pagination is always done in the controllers, as far as I can tell.